backend/api/serializers/performance.py

```diff
@@ -70,15 +77,17 @@ def get_round_idx(self, instance):
 
 
 class CPPerformanceSerializer(serializers.ModelSerializer):
-    compute_task = _PerformanceComputeTaskSerializer(read_only=True)
+    compute_task = _PerformanceComputeTaskSerializer(read_only=True, source="compute_task_output.task")
     metric = _PerformanceMetricSerializer(read_only=True)
```

Comment:
Do we still need `metric`? it looks like that we don't use any `metric` fields anymore.